script/build.js

```diff
@@ -205,6 +214,6 @@ if (options.watch) {
 }
 
 smith.build(function(err) {
-    if (err) throw err;
-    console.log('Build finished!');
-  });
+  if (err) throw err;
+  console.log('Metalsmith build finished!  Now starting webpack...');
```

Comment:
This would be better written as 'Now starting webpack-dev-server' for watch mode only.  I'll make that change...